templates/definition/meter/sofarsolar-g3.yaml

```diff
@@ -21,6 +21,8 @@ params:
     comset: 8N1
     port: 502
     id: 1
+  - name: delay
```

Comment:
Den Parameter kennen wir heute nicht. Damit das in configure schön wird fehlt vmtl. noch ein

    type: duration